symfony/symfony.lock

```diff
@@ -0,0 +1,281 @@
+{
```

Comment:
You don't commit lock here.

Reply:
This is not a production project, so I prefer every gets the latest versions of the dependencies.

Reply:
So same for composer.lock ?

Reply:
Exactly. 👍